src/go/util/marshal_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
 
+	statspb "github.com/envoyproxy/go-control-plane/envoy/config/metrics/v3"
```

Comment:
NIT: format import in dictionary order

Reply:
gofmt automatically sorts this back to the top. I think it's because it's sorting the import string alphabetically (config < extensions).